alws/utils/copr.py

```diff
@@ -22,10 +22,23 @@ def generate_repo_config(
 ) -> str:
     # we should clean "http" protocol from host url
     clean_host_name = re.sub(r'^(http|https)://', '', settings.pulp_host)
+    if 'debug-dr' in repo.url:
+        repo_url = re.sub(
+            '(x86_64|aarch64|ppc64le|i386|i686|s390x)-debug-dr',
+            '$basearch-debug-dr',
+            repo.url,
+        )
+    else:
+        repo_url = re.sub(
+            '(x86_64|aarch64|ppc64le|i386|i686|s390x)-dr',
+            '$basearch-dr',
+            repo.url,
+        )
+    repo_url = re.sub('almalinux-(8|9|10)', 'almalinux-$releasever', repo_url)
```

Comment:
I am not sure this regular expression will work 
[see](https://regex101.com/r/FLpjfT/1)

Reply:
This is the thing because some of the products were created outside of the community product creation workflow. It will replace things only for community products which is correct behavior